### PR TITLE
Wip/remove data

### PR DESCRIPTION
**Pull request by chrise96**

### CODE REVIEW

File: delete_blobs.Dockerfile [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# az acr build -t delete_blobs:latest -f delete_blobs.Dockerfile -r cvtweuacrogidgmnhwma3zq .
+FROM python:3.7.15-bullseye
```
Comment (epureanudiana) [inline delete_blobs.Dockerfile:2]: I see you decided not to use the PythonOperator and create the a KubernetesOperator. Is there a reason why you changed it?

Comment (chrise96) [reply delete_blobs.Dockerfile:2]: I wanted to use StorageAzureClient, this file is not in the cvt-airflow repo. Do you think PythonOperator is a cleaner solution?

File: submit_to_sia.py [hunk 2]
```diff
@@ -151,7 +152,9 @@ def _image_upload(auth_headers: Dict[str, str], filename: str, sig_id: str) -> A
     conn, cur = upload_to_postgres.connect()
 
     # Get images with a detection
-    sql = f"SELECT * FROM containers;"
+    sql = f"SELECT * FROM containers A LEFT JOIN images B ON A.closest_image = B.file_name " \
```
Comment (chrise96) [inline submit_to_sia.py:155]: TODO, what if all legal containers in teh database? Then we also select 10 items (LIMIT 1)

File: delete_blobs.py [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+from pathlib import Path
+
+from azure_storage_utils import StorageAzureClient
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "--stage",
+        type=str,
+        choices=["no_detections", "all_data"],
```
Comment (epureanudiana) [inline delete_blobs.py:12]: what are these choices?

File: delete_blobs.py [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+        print(
+            f"Removed {len(images_to_remove)} images without a detection from the cloud."
+        )
+
+    if opt.stage == "all_data":
+        # For a container, delete all content from a certain date.
+        cnames = ["blurred", "detections", "postprocessing-output"]
+
+        for cname in cnames:
+            # TODO what if empty?
```
Comment (epureanudiana) [inline delete_blobs.py:61]: it should not be empty, right? 
We have the slack operator which can tell us about this

File: submit_to_sia.py (hunk 2 above)
Comment (epureanudiana) [reply submit_to_sia.py:155]: We discussed about this today, right? We do something like `SELECT * FROM containers WHERE containers.score !=0`

File: delete_blobs.py [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+            cname="blurred",
+            blob_names=images_to_remove,
+        )
+        print(
+            f"Removed {len(images_to_remove)} images without a detection from the cloud."
+        )
+
+    if opt.stage == "all_data":
+        # For a container, delete all content from a certain date.
+        cnames = ["blurred", "detections", "postprocessing-output"]
```
Comment (epureanudiana) [inline delete_blobs.py:47]: nice

File: delete_blobs.py (hunk 4 above)
Comment (chrise96) [reply delete_blobs.py:50]: I'm interested if an empy Blob will break the code, I hope it will return [], but maybe it will return None

Comment (chrise96) [reply delete_blobs.py]: Fixed it in a nicer way now

File: submit_to_sia.py [hunk 6]
```diff
@@ -50,7 +51,7 @@ def _to_signal(date_now: date, lat_lng: Dict[str, float]) -> Any:
             }
         },
         "category": {
-            "sub_category": "/signals/v1/public/terms/categories/overlast-in-de-openbare-ruimte/sub_categories/overig-openbare-ruimte"  # TODO hinderlijk-geplaatst-object
+            "sub_category": "/signals/v1/public/terms/categories/overlast-in-de-openbare-ruimte/sub_categories/hinderlijk-geplaatst-object"
         },
         "reporter": {"email": "cvt@amsterdam.nl"},
```
Comment (epureanudiana) [inline submit_to_sia.py:56]: Let's look into this..whether we can get emails on this address, that's the address to which ASC reports back.

**Review by epureanudiana — APPROVED** [review]

**Pull request merged**